modules/lws-config.adoc

```diff
@@ -0,0 +1,129 @@
+//
+// * ai_workloads/leader_worker_set/lws-managing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lws-config_{context}"]
+= Deploying a leader worker set
+
+You can use the {lws-operator} to deploy a leader worker set to assist with managing distributed workloads across nodes.
+
+// TODO: check config or default differences: https://redhat-internal.slack.com/archives/C087LPW0VDZ/p1755502508848329
+
+.Prerequisites
+
+* You have installed the {lws-operator}.
+// TODO anything else?
+
+.Procedure
+
+. Create a new project by running the following command:
++
+[source,terminal]
+----
+$ oc new-project my-namespace
+----
+
+. Create a file named `leader-worker-set.yaml`
++
+[source,yaml]
+----
+apiVersion: leaderworkerset.x-k8s.io/v1
+kind: LeaderWorkerSet
+metadata:
+  generation: 1
+  name: my-lws <1>
+  namespace: my-namespace <2>
+spec:
+  leaderWorkerTemplate:
+    leaderTemplate: <3>
+      metadata: {}
+      spec:
+        containers:
+        - image: nginxinc/nginx-unprivileged:1.27
+          name: leader
+          resources: {}
+    restartPolicy: RecreateGroupOnPodRestart <4>
+    size: 3 <5>
+    workerTemplate: <6>
+      metadata: {}
+      spec:
+        containers:
+        - image: nginxinc/nginx-unprivileged:1.27
+          name: worker
+          ports:
+          - containerPort: 8080
+            protocol: TCP
+          resources: {}
+  networkConfig:
+    subdomainPolicy: Shared <7>
+  replicas: 2 <8>
+  rolloutStrategy:
+    rollingUpdateConfiguration:
+      maxSurge: 1 <9>
+      maxUnavailable: 1
+    type: RollingUpdate
+  startupPolicy: LeaderCreated
+----
+<1> Specify the name of the leader worker set resource.
+<2> Specify the namespace for the leader worker set to run in.
+<3> Specify the pod template for the leader pods.
+<4> Specify the restart policy for when pod failures occur. Allowed values are `RecreateGroupOnPodRestart` to restart the whole group or `None` to not restart the group.
+<5> Specify the number of pods to create for each group, including the leader pod. For example, a value of `3` will create 1 leader pod and 2 worker pods. The default value is `1`.
+<6> Specify the pod template for the worker pods.
+<7> Specify the policy to use when creating the headless service. Allowed values are `UniquePerReplica` or `Shared`. The default value is `Shared`.
+<8> Specify the number of replicas, or leader-worker groups. The default value is `1`.
+<9> Specify the maximum number of replicas that can be scheduled above the `replicas` value during rolling updates. The value can be specified as an integer or a percentage.
++
+For more information on all available fields to configure, see link:https://lws.sigs.k8s.io/docs/reference/leaderworkerset.v1/[LeaderWorkerSet API] upstream documentation.
++
+// Do we need to list the character limit for LWS name here or as a known issue? https://lws.sigs.k8s.io/docs/troubleshooting/#3-unable-to-create-lws-object-with-a-name-exceeding-51-characters
```

Comment:
@ardaguclu Since this is listed as a known issue for LWS, should we mention here about this character limitation for the LWS name?

Reply:
I think, we already reference the upstream documentation from our documentation (which mentions this issue). So maybe we can remove this. Otherwise, the expectation would be listing all the known issues in here.

Reply:
Okay thanks, I won't worry about it.